### DISCUSSION TITLE
매퍼에서 자바 코드로 매핑

### DISCUSSION THREAD

**Discussion by Dae-Hwa** [Sc0up/scoup-backend#94] [Refactor]
https://github.com/Sc0up/scoup-backend/pull/84#discussion_r693718460

**Comment by Dae-Hwa** [comment]
현재는 필요 없다고 판단. 추후에 꼭 필요하면 다시 논의

**Comment by janeljs** [comment]
Entity에 컬렉션이 있는 경우
- https://www.baeldung.com/java-mapstruct-mapping-collections

**Reply by janeljs** [reply]
### custom mapper
https://www.baeldung.com/mapstruct-custom-mapper

**Reply by janeljs** [reply]
https://mapstruct.org/documentation/stable/reference/html/

**Reply by janeljs** [reply]
### multiple fields
https://stackoverflow.com/questions/37705417/map-multiple-source-fields-to-same-type-target-fields-with-mapstruct

**Reply by janeljs** [reply]
### after mapping
https://stackoverflow.com/questions/59028797/aftermapping-is-not-called-from-mapper-interface

**Comment by Dae-Hwa** [comment]
Before Mapping & After Mapping

**Reply by Dae-Hwa** [reply]
https://mapstruct.org/documentation/stable/reference/html/#customizing-mappings-with-before-and-after
